network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_double_image_block.html

```diff
@@ -1,37 +1,38 @@
 {% load wagtailcore_tags wagtailimages_tags remove_richtext_wrapping %}
 
-</div> <!-- End .col-sm-10 from main container -->
-  <div class="col-sm-10 offset-sm-1 double-image-block">
-    <div class="row">
-      <div class="col-sm-6">
+<div class="container {{block.block_type}}-block">
+  <div class="row justify-content-center">
+    <div class="streamfield-content">
+      <div class="double-image-block">
+        <div class="row">
+          <div class="col-sm-6">
 
-        {% image self.image_1 fill-445x324 as img1 %}
-        <figure class="my-default">
-          <img src="{{ img1.url }}" alt="{{ img1.alt }}" class="w-100" />
-          {% if self.image_1_caption %}
-            <figcaption>
-              {{ self.image_1_caption|richtext|remove_wrapping|safe }}
-            </figcaption>
-          {% endif %}
-        </figure>
+            {% image self.image_1 fill-445x324 as img1 %}
```

Comment:
I wonder if we should use [dynamic images](https://docs.wagtail.io/en/stable/advanced_topics/images/image_serve_view.html) here to speed up page renditions while we're at it?

Reply:
Hi Kalob, submitted issue #6921 to implement this in the future!